webpack/prod.config.js

```diff
@@ -0,0 +1,10 @@
+const merge = require('webpack-merge');
+const baseConfig = require('./base.config');
+
+module.exports = merge(baseConfig, {
+  mode: 'production',
+  devtool: 'none',
+  optimization: {
+    minimize: true
+  }
```

Comment:
Might be a good idea to check whether production mode is enough to infer optimisation settings.